docs/api/event-brokers.rst

```diff
@@ -231,3 +235,51 @@ according to the security protocol being used. The following implementation show
 
     for message in consumer:
         print(message.value)
+
+SQL Event Broker
+-----------------
+
+It is possible to use  an SQL database as an event broker. Connections to databases are established using
+`SQLAlchemy <https://www.sqlalchemy.org/>`_, a Python library which can interact with many
+different types of SQL databases, such as `SQLite <https://sqlite.org/index.html>`_,
+`PostgreSQL <https://www.postgresql.org/>`_ and more. The default Rasa installation allows connections to SQLite
```

Comment:
> The default Rasa installation allows connections to SQLite
and PostgreSQL databases

Not getting this comment. We also allow others, don't we? It's just not tested by us.

Reply:
I added in this comment because Rasa depends on `psycopg2`. The Python stdlib already bundles a SQLite driver. So when you install Rasa, by default, you're only able to use those two databases, unless you explicitly install a new driver.